index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
@harshvitra I don't think we need this file here. Do you want to check variants.html?